### ISSUE TITLE
IF YOU HAVE ANY WORKING METHODS POST HERE

### ISSUE THREAD

**Issue by aztro-leakz** [aztro-leakz/BYPAZZR#1]
Im Trying my best to find these methods  and I need help. Plz post any working securly bypassed yall know Thx :3

**Comment by Agguiopz** [comment]
[Here](https://docs.google.com/document/d/11PT9roT8YTfLQEFPgCDdrf8IXFqh_XXEjgWcCywXW6c/edit?tab=t.0)
I recommend checking [ext-remover](https://ext-remover.net)
They have pretty good methods

**Comment by tacticalarms1** [comment]
rigtools got patched in v129 so u can remove that

**Comment by aztro-leakz** [comment]
> rigtools got patched in v129 so u can remove that

@tacticalarms1 well first off a lot of ppl arent on v129 including me so I wont  remove it. second I'm trying to find something that works just  like rig tools but on all versions

**Comment by tacticalarms1** [comment]
recently I found static proxy

**Comment by aztro-leakz** [comment]
> recently I found static proxy

@tacticalarms1 would you like to share it with me? depending on if it works I can put it in the doc and credit u for finding it

**Comment by tacticalarms1** [comment]
sure if but I have not tested it but I have seen the test o it probably works.

**Comment by aztro-leakz** [comment]
uh okay well if its a GitHub repo or smth send it I'll check it out

**Comment by tacticalarms1** [comment]
my school uses lightspeed so I need an unblocker that works longer than ext hanger

**Comment by tacticalarms1** [comment]
bc I wanna play fortnite

**Comment by aztro-leakz** [comment]
well

**Comment by aztro-leakz** [comment]
do you have access to gihub?

**Comment by aztro-leakz** [comment]
GitHub codespace?

**Comment by tacticalarms1** [comment]
no but I can unblock using exit hanger

**Comment by tacticalarms1** [comment]
here's the link for nano demo
https://nano-proxy.github.io/

**Comment by tacticalarms1** [comment]
no proxy functionality but once I finish build then I can host on GitHub pages'

**Comment by tacticalarms1** [comment]
gotta head to next period

**Comment by aztro-leakz** [comment]
I already have that listed in the doc. I added it like 2 minutes ago lol

**Comment by tacticalarms1** [comment]
can u credit for finding also wanna collab on this together?

**Comment by tacticalarms1** [comment]
btw what blocking software does your school use?

**Comment by aztro-leakz** [comment]
> btw what blocking software does your school use?

securly. if you read  the doc it says all of these methods are tested on securly

**Comment by aztro-leakz** [comment]
if your looking for long lasting methods to  play fortnight. try the geobra bypass I think its number 10. if that doesn't work I recommend dextensify impporved which is like 9 and then If that don't work try rig tools. unless ur chromebook is v129

**Comment by aztro-leakz** [comment]
@tacticalarms1

**Comment by aztro-leakz** [comment]
> no proxy functionality but once I finish build then I can host on GitHub pages'

yeah how would you do smth like that I've been trying to find out @tacticalarms1

**Comment by Agguiopz** [comment]
> hey @Agguiopz I got a question abt rig tools

Sorry but I have never deployed rigtools and if you can't deploy it then nobody probably can.

**Comment by aztro-leakz** [comment]
Ah well I'll figure it out. I've seen many ppl deploy it so I'm hoping it's not too hard

**Comment by tacticalarms1** [comment]
oh yeah and about the geobra bypass u can just go to chrome://sign-in but it does not work with fortnite because you cant Fullscreen so it will not let you play

**Comment by tacticalarms1** [comment]
if u need anything just ask

**Comment by tacticalarms1** [comment]
o and I ported paper Minecraft so here's the link
tacticalarms1.github.io/papermc

**Comment by tacticalarms1** [comment]
I might just make my own bypass list like this and ext remover and the others

**Comment by tacticalarms1** [comment]
its just if my friends find out they will take all my exploits

**Comment by aztro-leakz** [comment]
> oh yeah and about the geobra bypass u can just go to chrome://sign-in but it does not work with fortnite because you cant Fullscreen so it will not let you play

That's blocked for me

**Comment by aztro-leakz** [comment]
> I might just make my own bypass list like this and ext remover and the others

I'm willing to work with you. I can give you editor on the doc ect but I will have to ask you a few things.

**Comment by tacticalarms1** [comment]
oh and ig u don't know how to host to GitHub pages' so I can help you make a website If you want

**Comment by aztro-leakz** [comment]
the thing with GitHub pages is its blocked for  me. same with many other ppl. BUT I do like web design but I know like no code. but I  have made many websites design before so I'm down to make a site I'm very new to pages though. I was up all last night reading abt em but  still is confusing to me

**Comment by aztro-leakz** [comment]
@tacticalarms1

**Comment by tacticalarms1** [comment]
so u wanna do it or no
also I made a team so we can invite other people to help

**Comment by aztro-leakz** [comment]
uh sure? just fyi I'm not very experienced with github and I only know so much lol

**Comment by tacticalarms1** [comment]
check ur notifications I invited u

**Comment by aztro-leakz** [comment]
uh okey

**Comment by aztro-leakz** [comment]
I'm in art so I cant talk much sry

**Comment by aztro-leakz** [comment]
I don't see anything in my inbox

**Comment by tacticalarms1** [comment]
oh that's weird

**Comment by aztro-leakz** [comment]
wait in GitHub or Google? I checked github

**Comment by tacticalarms1** [comment]
go here https://github.com/BYPAZZR

**Comment by aztro-leakz** [comment]
okey

**Comment by aztro-leakz** [comment]
wuts do I  do there?

**Comment by aztro-leakz** [comment]
ohhhh

**Comment by aztro-leakz** [comment]
sick

**Comment by aztro-leakz** [comment]
get it

**Comment by tacticalarms1** [comment]
il give u owner

**Comment by aztro-leakz** [comment]
uh okey

**Comment by tacticalarms1** [comment]
did u accept?

**Comment by aztro-leakz** [comment]
I'm now a member?

**Comment by aztro-leakz** [comment]
I saw nothing abt owner

**Comment by tacticalarms1** [comment]
gtg for a bit

**Comment by aztro-leakz** [comment]
aighty

**Comment by aztro-leakz** [comment]
uh who made the pfp for it. cuz it says chat gpt 😭 we might need to change that

**Comment by tacticalarms1** [comment]
lol

**Comment by aztro-leakz** [comment]
uh yeah I need a better logo tbh the bypazzrr logo was supposed to give the doc more of a "tool" kind of look you know what I mean if u have ever used like a ddos tool or like another tool that has a logo

**Comment by aztro-leakz** [comment]
but its mid asf

**Comment by aztro-leakz** [comment]
Ill make one or smth

**Comment by tacticalarms1** [comment]
ok

**Comment by tacticalarms1** [comment]
https://github.com/BYPAZZR/bypazzr.github.io
our "main" site il host after we are done making it

**Comment by tacticalarms1** [comment]
I made one I'm adding finishing touches

**Comment by tacticalarms1** [comment]
![E (1)](https://github.com/user-attachments/assets/19db5682-548b-4ee4-8eec-2b07420fa16c)

**Comment by aztro-leakz** [comment]
I believe EXT BYPASS is taken that's why I named it BYPAZZR just fyi

**Comment by aztro-leakz** [comment]
what aplication or site did u use?

**Comment by tacticalarms1** [comment]
oh dang I live about 2 hours from u

**Comment by aztro-leakz** [comment]
crazy

**Comment by aztro-leakz** [comment]
anyone can dox me bc of my email lol IDC thro

**Comment by tacticalarms1** [comment]
lol

**Comment by tacticalarms1** [comment]
gtg lunch

**Comment by aztro-leakz** [comment]
k

**Comment by aztro-leakz** [comment]
ur lunch over yet?

**Comment by aztro-leakz** [comment]
I made the read me

**Comment by tacticalarms1** [comment]
yeah
ok

**Comment by aztro-leakz** [comment]
@tacticalarms1 u gotta chill. u changing everything to "we" and like  that's fine and  all but I ain't trying to get stolen from or have my credit stolen so chill cuz that shit happened to me a while back. once you like add some methods then that's fine but like don't be tryna steal credit yk. not saying u are js making sure you wonk ya feel me?

**Comment by aztro-leakz** [comment]
sorry if that's like random or I'm getting the wrong idea it's js I had like a discord server with over 1k members get taken from me so that's why I said all at

**Comment by tacticalarms1** [comment]
Ok mb

**Comment by tacticalarms1** [comment]
I'm not trying steal credit I'm super honorable

**Comment by tacticalarms1** [comment]
oh yeah and ik how u feel it happened to me quite a few times before
